docs: add how to extend the web (#114)
equinor/template-fastapi-react: documentation/docs/about/01-introduction.md

This is a solution template for creating a Single Page App (SPA) with React and FastAPI following the principles of Clean Architecture.

> The template provides an example implementation of a [todo application](https://template-fastapi-react.app.radix.equinor.com/). The todo app implemented is pretty basic. A user can add a task, mark a task as completed and delete an already added task. The purpose of the minimalist todo app implementation are to learn and practice what the concepts of [clean architecture](../contribute/development-guide/coding/01-architecture.md) are and how it can be used in a REST API. 

Reviewer: ```suggestion
The template provides an example implementation of a [todo application](https://template-fastapi-react.app.radix.equinor.com/). The todo app implementation is fairly basic. A user can add a task, mark a task as completed and delete an added task. The purpose of the minimalist todo app implementation is to learn and practice what the concepts of [clean architecture](../contribute/development-guide/coding/01-architecture.md) are, and how they can be used in a REST API. 
```

Author: fixed
